scripts/run-druid

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
```

Comment:
This script lives in the docker-images repo now, doesn't it ? I think it should be removed here.

Reply:
Yes, I removed it, thank you